frontend/packages/dev-console/src/components/deployments/DeploymentForm.tsx

```diff
@@ -53,8 +53,12 @@ const EditDeploymentForm: React.FC<
 
   const yamlEditor = (
     <>
-      <DeploymentConfigDeprecationAlert />
-      <br />
+      {resource.kind === DeploymentConfigModel.kind && (
+        <>
+          <DeploymentConfigDeprecationAlert />
+          <br />
```

Comment:
No, need to add `br` here it looks better without it and gives the user more space on the screen for the editor. See the screenshot below.

<img width="1222" alt="image" src="https://github.com/openshift/console/assets/2561818/d917d1b8-3716-40b4-8b5c-904180a6ff0c">

vs 

<img width="1222" alt="image" src="https://github.com/openshift/console/assets/2561818/6df435e6-c0a1-467f-9e26-60cfed21fa34">

Reply:
Thanks. Updated the PR.